packages-exp/installations-exp/src/api/delete-installations.ts

```diff
@@ -23,6 +23,8 @@ import { FirebaseInstallationsImpl } from '../interfaces/installation-impl';
 import { FirebaseInstallations } from '@firebase/installations-types-exp';
 
 /**
+ * Deletes the Firebase Installation and all associated data.
+ *
  * @public
```

Comment:
what does this line mean?

Reply:
The `@public` is the annotation that used by the tool `api-extractor` for auto-generating public docs.